components/ADempiere/PanelInfo/Component/RecordIssues/component/Comment.vue

```diff
@@ -1012,13 +1012,16 @@ export default defineComponent({
       })
     }
 
-    function addNewComments(params) {
+    function addNewComments() {
       const { id, uuid } = currentIssues.value
       store.dispatch('newComments', {
         id,
         uuid,
         result: comments.value
       })
+        .then(response => {
+          refs.scrollTimeLineTabComments.$refs.wrap.scrollTop = 9999999
```

Comment:
please remove `refs.scrollTimeLineTabComments`, it is obsolete, change to
```javascript
const scrollTimeLineTabComments = ref(null)
```

and add nextTick on response to prevent rendering errors
```javascript
.then(response => {
  nextTick(() => {
    scrollTimeLineTabComments.value.$refs.wrap.scrollTop = 9999999
  })
})
```